packages/paste-website/src/pages/components/list/index.mdx

```diff
@@ -0,0 +1,237 @@
+  {`<UnorderedList>
+  <ListItem>
+    Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. <em>Curabitur blandit tempus</em> porttitor. Nulla vitae elit libero, a pharetra augue. Donec id elit non mi porta gravida at eget metus.
+  </ListItem>
+  <ListItem>
+    Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nulla vitae elit libero, a pharetra augue.
+  </ListItem>
+</UnorderedList>`}
+</LivePreview>
+
+### Ordered List
+
+Use ordered list to display a numbered list of items.
+
+<LivePreview scope={{OrderedList, ListItem}} language="jsx">
+  {`<OrderedList>
+  <ListItem>
+    Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. <em>Curabitur blandit tempus</em> porttitor. Nulla vitae elit libero, a pharetra augue. Donec id elit non mi porta gravida at eget metus.
+  </ListItem>
+  <ListItem>
+    Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nulla vitae elit libero, a pharetra augue.
+  </ListItem>
+</OrderedList>`}
+</LivePreview>
+
+### Nesting list items
+
+List items can be nested within other list items. To do so create a new list as a child to a `ListItem`.
+
+<LivePreview scope={{OrderedList, UnorderedList, ListItem}} language="jsx">
+  {`<div>
+  <OrderedList>
+    <ListItem>
+      Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
+      <UnorderedList>
+        <ListItem>
+          Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
+        </ListItem>
+        <ListItem>
+          Duis mollis, est non commodo luctus.
+        </ListItem>
+      </UnorderedList>
+    </ListItem>
+    <ListItem>
+      Duis mollis, est non commodo luctus.
+    </ListItem>
+  </OrderedList>
+  <UnorderedList>
+    <ListItem>
+      Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
+      <OrderedList>
+        <ListItem>
+          Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
+        </ListItem>
+        <ListItem>
+          Duis mollis, est non commodo luctus.
+        </ListItem>
+      </OrderedList>
+    </ListItem>
+    <ListItem>
+      Duis mollis, est non commodo luctus.
+    </ListItem>
+  </UnorderedList>
+</div>`}
+</LivePreview>
+
+## Composition Notes
+
+### When to Use a OrderedList and UnorderedList
+
+<DoDont>
+  <Do title="Do" body="Use a list when you have more than one item or point." center>
+    <UnorderedList>
+      <ListItem>Point one</ListItem>
+      <ListItem>Point two</ListItem>
+      <ListItem>Point three</ListItem>
+    </UnorderedList>
+  </Do>
+  <Dont title="Don't" body="Don’t use lists for a single item." center>
+    <UnorderedList>
+      <ListItem>Point one</ListItem>
+    </UnorderedList>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use nested lists to provide additional clarity or instruction on specific topics." center />
```

Comment:
We should always provide examples for DoDonts imo, otherwise the DoDont component looks bad and the information would fit under the normal guidelines section.

Reply:
updated